sys/include/riotboot/flashwrite.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2018 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_riotboot_flashwrite riotboot flash writer
```

Comment:
*R*iotboot?

Reply:
IMO all the riotboot module documentation needs work. I suggest we postpone that to a later PR.

Reply:
:+1: